scripts/variantstore/gvs-overview.md

```diff
@@ -0,0 +1,165 @@
+
+**3.** Grant your proxy group the **BigQuery Data Editor** role on the dataset you created in Step 1 by following the instructions in [Controlling access to datasets](https://cloud.google.com/bigquery/docs/dataset-access-controls) from the BigQuery documentation.
+
+**4.** Grant your proxy group the **BigQuery Data Editor**, **BigQuery Job User**, and **BigQuery Read Session User** roles on the Google Project that contains the dataset you created in Step 1 by following the instructions in [Manage access to projects, folders, and organizations](https://cloud.google.com/iam/docs/granting-changing-revoking-access) from the Google Cloud documentation.
+
+#### Reblocked gVCF files
+
+The GVS workflow takes in reblocked gVCF files as input. If your files are not already reblocked, you can reblock them using the [WARP reblocking workflow](https://github.com/broadinstitute/warp/blob/master/pipelines/broad/dna_seq/germline/joint_genotyping/reblocking/ReblockGVCF.wdl), which is configured in the [ReblockGVCF Terra workspace](https://app.terra.bio/#workspaces/warp-pipelines/ReblockGVCF). For more information about reblocking, check out [WARP Whole Genome and Exome Pipelines Produce Reblocked GVCFs](https://broadinstitute.github.io/warp/blog/tags/reblock/).
+
+#### gVCF annotations
+
+Input gVCF files for the GVS workflow must include the annotations described in the table below:
+
+| Annotation | Description | Notes |
+| --- | --- | --- |
+| Ref | Reference allele | --- |
+| Alt | Alternate allele | --- |
+| AS_RAW_MQ, RAW_MQandDP, or RAW_MQ | RMS mapping quality (‘AS’: allele-specific) | Required for VQSR Data |
+| AS_RAW_MQRankSum or Map_QUAL_RANK_SUM_KEY | Z-score from Wilcoxon rank sum test of alternate versus reference read mapping qualities | Required for VQSR Data |
+| QUALapprox | Sum of PL[0] values; used to approximate the QUAL score | Required for VQSR Data |
+| AS_QUALapprox | Allele-specific sum of PL[0] values; used to approximate the QUAL score | Required for VQSR Data |
+| AS_SB_TABLE or STRAND_BIAS_BY_SAMPLE | Allele-specific forward/reverse read counts for strand bias tests | Required for VQSR Data |
+| AS_VarDP, VarDP, or DP | Depth over variant genotypes, or read depth  (‘AS’: allele-specific) | Required for VQSR Data |
+| call_GT | Genotype | --- |
+| call_GQ | Genotype quality | --- |
+
+### Inputs
+
+The GVS workflow inputs are specified in the Terra GVS beta workspace [workflow configuration](LINK).
+
+#### Sample data input
+
+The GVS workflow takes in reblocked single sample gVCF files and their corresponding index files as `input_vcfs` and `input_vcf_indexes`, respectively. While the GVS workflow takes in up to 100,000 single sample gVCF files as input, only datasets of up to 10,000 files are being used for beta testing.
```

Comment:
Two things

1. as `input_vcfs` -- I think it's implied here that we're talking about the terra data model?  Maybe make that more clear esp if the audience is new to Terra

2. Again I would remove the 100k piece and just say "For the beta, cohort sizes of up to 10k samples are supported"